packages/cosmos/src/error.rs

```diff
@@ -94,7 +95,7 @@ pub enum BuilderError {
 }
 
 /// Parse errors while interacting with chain data.
-#[derive(thiserror::Error, Debug, Clone)]
```

Comment:
I'm fairly certain we rely on this `Clone` elsewhere. Regardless, i don't want to lose the ability to clone for this case. It's also surprising to me that we're getting 3 new variants here to represent one feature. And the variants don't give any indication of what we were trying to do that failed.

I'm not sure which piece of my request led to the three extra variants and removing the `Clone`, but I'd probably go in the direction of a single, cloneable error variant for all ways the EIP parsing could fail.

Reply:
> And the variants don't give any indication of what we were trying to do that failed.

Isn't that encoded in the `action` part of `Error::ChainParse` ?

> I'm not sure which piece of my request led to the three extra variants and removing the Clone

The application code consolidates it all into an `anyhow::Error`. The library code does not, and from what I see, being specific with these different types is a pattern followed in different places in the codebase.

How would you suggest we keep the information about `StdError` (which doesn't impl `Clone`)?

One idea is to just toss out the detailed info error and just have a `TxFeesError` variant without the inner data. Is that ok?

Reply:
I’m fine with tossing out details. But as a middle position, how about a single variant that includes a string field that has the original error value rendered to string?

Reply:
works for me, will do that now

Reply:
https://github.com/Levana-Protocol/levana-cosmos-rs/pull/69/commits/8f878612406b41e8083ea706b3bd61400f4754a4 look ok?

Reply:
LGTM